config/runtime.exs

```diff
@@ -6,7 +6,8 @@ source(["config/.env", "config/.env.#{config_env()}"])
 if config_env() == :prod do
   config :dbservice, Dbservice.Repo,
     url: env!("DATABASE_URL", :string!),
-    pool_size: env!("POOL_SIZE", :integer) || 10
+    pool_size: env!("POOL_SIZE", :integer) || 10,
+    ssl: true
```

Comment:
what is this used for?

Reply:
I encountered a "no encryption" issue for the db connection during app startup, which required me to include the config to enable encryption for data in transit.